app/theme/client/imports/components/header.css

```diff
@@ -413,7 +413,7 @@
 @media (width <= 500px) {
 	.rc-header {
 		&__visual-status {
-			display: none;
+			font-size: 100%;
```

Comment:
What effect does `font-size: 100%` has here? Is there any specific need for it?

Reply:
Now I see that `font-size: 100%;` is not required. I have removed the unnecessary code. Is it fine now?